### ISSUE TITLE
Show 'complete' output from observanble

### ISSUE THREAD

**Issue by niklasR** [SamVerschueren/listr#72]
Hello,

I use `zen-observable`, and the `.next()` to update the progress. At the end of the task I would either like to
 - persist the last output from `.next()`
 - or be able to update the task name by some means (i.e. passing a resolution to `.complete()`)

My use case is going through a number of files looking for a 'match', and it'd be nice if I could print on the console what the match is for each task.

**Comment by sompylasar** [comment]
Yes, this would be very helpful for tasks that output something as a result which has to be persisted for the user to read.

**Comment by sompylasar** [comment]
Related:
- https://github.com/SamVerschueren/listr-update-renderer `collapse` option
- https://github.com/SamVerschueren/listr/issues/73

**Comment by SamVerschueren** [comment]
This could be an extra option in the `update-renderer`. Not sure about the name though, but suggestions and PR welcome.

**Comment by filiptc** [comment]
Big +1 for this. Name could be in the lines of `persistLastOutput`. I'll try to squeeze a PR in if I have some spare time.
